ExpenseManagerUsers/Startup.cs

```diff
@@ -26,6 +26,32 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
+            services.Configure<Dbconfig>(Configuration.GetSection("ConnectionStrings"));
+
+            services.AddSingleton<DbContext>(provider =>
+            {
+                string connectionString = provider.GetRequiredService<IOptions<Dbconfig>>().Value.DefaultConnection;
+                return new DbContext(connectionString);
+            });
+
+            services.AddSingleton<IUserService, UserService>();
+
+            services.Configure<JwtConfig>(Configuration.GetSection("TokenOptions"));
+
+            services.AddSingleton<ITokenService, TokenService>();
+
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)

```

Comment:
Is the jwt stuff needed in user service? If so, this looks the same to the transaction service, worth sharing the code. Please also look for other duplicated code between the two services which is stable (not likely to change in the future), try to share them if possible.

Reply:
removed JWT bearer and Authentication from Users endpoint as it's not being used